SECURITY.md

```diff
@@ -0,0 +1,13 @@
+# Security Policy
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 4.0.x   | :white_check_mark: |
+| < 4.0   | :x:                |
+
+
+## Reporting a Vulnerability
+
+Patrick will tell us what to put here.
```

Comment:
It might be good to have this info in the Sphinx project and just link to that from this file so GitHub knows we have a security policy.